libr/main/rabin2.c

```diff
@@ -893,6 +893,13 @@ R_API int r_main_rabin2(int argc, const char **argv) {
 		return 1;
 	}
 	file = argv[opt.ind];
+
+	if (file && file[0] == '\0') {
```

Comment:
Use R_STR_IS_EMPTY

Reply:
Please see discussion in https://github.com/radareorg/radare2/pull/17126/files#r443814345